src/task/cleanup.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Friemt\Deployer\Tasks;
+
+use function Deployer\get;
+use function Deployer\run;
+use function Deployer\set;
+use function Deployer\task;
+use function Deployer\test;
+use function Deployer\within;
+use function Deployer\writeln;
+
+set('cleanup_paths', ['var/cache']);
+
+task('cleanup:paths', function (): void {
+    $currentRelease = basename(get('release_or_current_path'));
+    $releases = get('releases_list');
+    $sudo = get('cleanup_use_sudo') ? 'sudo' : '';
```

Comment:
`sudo` may not work on all *nix system, use a string variable instead of a boolean and set `sudo` as default value.

Reply:
I agree. However, [deployer does it the same way](https://github.com/deployphp/deployer/blob/a52ae3da6a4bcf10f7bf42f7fac7d982a62240c2/recipe/deploy/cleanup.php#L11) and I thought I'd stick to that.